lib/common.sh

```diff
@@ -213,6 +229,21 @@ resolve_ipv4()
         return 1
     fi
 
+    #
+    # Check if the IP-FQDN pair is present in /etc/hosts
+    # 
+    if is_present_in_etc_hosts "$ipv4_addr" "$hname"; then
+        if [ "$fail_if_present_in_etc_hosts" == "true" ]; then
+            eecho "[FATAL] Detected entry $ipv4_addr $hname in /etc/hosts."
+            eecho "$hname is resolving into $ipv4_addr from /etc/hosts."
+            eecho "Remove the entry for "$hname" in /etc/hosts to ensure proper handling of endpoint IP address changes by AZNFS."
+            return 1
+        else
+            wecho "[FATAL] Detected entry $ipv4_addr $hname in /etc/hosts." 1>/dev/null
+            wecho "Remove the entry for "$hname" in /etc/hosts to ensure proper handling of endpoint IP address changes by AZNFS." 1>/dev/null
```

Comment:
remove dot at the end of each log, we don't add dot at the end of logs in other places, isn't it?
also why do you want to suppress log coming on stdout at L242 243, let it come on stdout

Reply:
Here, I have suppressed the this log, as otherwise value of new_ip in aznfswatchdog gets updated to this error log
L 315:  new_ip=$(resolve_ipv4 "$l_host" "false").